scripting/get5.sp

```diff
@@ -846,8 +846,7 @@ public Action Command_Stop(int client, int args) {
         return Plugin_Handled;
     }
 
-    int roundsPlayed = GameRules_GetProp("m_totalRoundsPlayed");
-    if (g_GameState != GameState_Live || roundsPlayed == 0) {
+    if (g_GameState != GameState_Live || g_PendingSideSwap == true ) {
```

Comment:
I can't say I tested that with the new updated code, did it before and you couldn't do it on the first round, let me take a look.